pkg/apis/core/v1alpha1/types.go

```diff
@@ -219,18 +229,25 @@ const DataVolumeCloneSourceSubresource = "source"
 // CDI is the CDI Operator CRD
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=cdi;cdis,categories=all,scope=Cluster
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase"
 type CDI struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   CDISpec   `json:"spec"`
-	Status CDIStatus `json:"status"`
+	Spec CDISpec `json:"spec"`
+	// +optional
+	Status CDIStatus `json:"status,omitempty"`
```

Comment:
not sure `omitempty` makes sense for a non pointer struct member

Reply:
Good point, the optional does the trick, same in the betav1